app/controllers/actions/AuthAction.scala

```diff
@@ -39,13 +39,13 @@ class AuthActionImpl @Inject()(override val authConnector: AuthConnector, config
   val ctUtr: Retrieval[Option[String]] = OptionalRetrieval("ctUtr", Reads.StringReads)
 
   private[controllers] def getEnrolment(enrolments: Enrolments): CtEnrolment = {
-    enrolments.getEnrolment("IR-CT")
-      .map(e => CtEnrolment(CtUtr(e.getIdentifier("UTR").map(_.value).getOrElse(throw new UnauthorizedException("Unable to retrieve ct utr"))), e.isActivated))
-      .getOrElse(throw new UnauthorizedException("unable to retrieve ct enrolment"))
+    enrolments.getEnrolment("HMCE-VATDEC-ORG")
```

Comment:
Could this be a val declared in the class? Could make it easier to update AuthActionfor future frontends that are based on vat-frontend?